### PR TITLE
SOF-1959: Setup barebone package structure

### PR DESCRIPTION
**Pull request by RasmusAlbrektsen**
Basic package structure, testing, linting setup that is very much alike to our setup on Sofie TV2 Server. However, for this repository i have added Jest as a testing framework, since i did not know whether Vitest was something we're just trying out on a single repository?
Eslint used for linting.

### CODE REVIEW

**Review by aardhuis — APPROVED** [review]
Nice :)

File: package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "inews-gateway",
+  "version": "0.0.0",
+  "repository": "https://github.com/tv2/inews-gateway.git",
+  "contributors": [
+    {
+      "name": "Rasmus Emil Albrektsen",
+      "email": "<raal@tv2.dk>"
+    }
+  ],
```
Comment (MadsThy) [inline package.json:10]:
```suggestion
```

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "outDir": "dist",
+
+    "sourceMap": false,
+    "removeComments": true,
+    "resolveJsonModule": true,
+    "forceConsistentCasingInFileNames": true,
+
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+
```
Comment (MadsThy) [inline tsconfig.json:18]:
```suggestion
    "rootDir": "src",
    "outDir": "dist",
    "sourceMap": false,
    "removeComments": true,
    "resolveJsonModule": true,
    "forceConsistentCasingInFileNames": true,
    "experimentalDecorators": true,
    "emitDecoratorMetadata": true,
```

**Review by MadsThy — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (RasmusAlbrektsen) [reply package.json]: Removed!

File: tsconfig.json (hunk 2 above)
Comment (RasmusAlbrektsen) [reply tsconfig.json]: I have removed the newlines :-)

**Review by LindvedKrvang — APPROVED** [review]

**Pull request merged**